### PR TITLE
MTV-674: release notes for MTV 2.5.1

### PR DESCRIPTION
**Pull request by anarnold97**
@ahadas - please can you list what you want covering in the release notes

I thought it best to give you access so you can add notes

Thanks

### CODE REVIEW

File: documentation/modules/rn-2.5.adoc [hunk 1]
```diff
@@ -124,18 +124,14 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
 
 When adding an OVA provider, the error message `ConnectionTestFailed` may instantly appear, although the provider is created successfully. If the message does not disappear after a few minutes and the provider status does not move to `Ready`, this means that the `ova server pod creation` has failed. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
 
-.OVA migration with multiple disks does not function as expected
```
Comment (ahadas) [inline documentation/modules/rn-2.5.adoc:127]: @anarnold97 I've just dropped this one as it's been an issue for just ~a month and we didn't get reports on that

**Review by ahadas — COMMENTED** [review]
@anarnold97 it was easier for me to add another commit to this PR
those are all the changes needed in the release notes for 2.5.1

**Comment by anarnold97** [comment]
@ahadas - thanks for your help. I will keep the ticket open until early next week, then I will do a re-write of any notes you add.

Thanks

File: documentation/modules/rn-2.5.adoc [hunk 2]
```diff
@@ -196,18 +192,11 @@ It is recommended to upgrade from {project-short} 2.4.2 to {project-short} {proj
...
-====
-There is an issue with upgrading from 2.4.1 or 2.4.2.
-When upgrading from 2.4 (2.4.z), after the 2.5 operator starts
+[id="deprecated-features-25_{context}"]
+== Deprecated features
 
-. Remove the `mutatingwebhookconfiguration` named `forklift-api` by running the following command:
-+
-`$ oc -n openshift-mtv delete mutatingwebhookconfiguration forklift-api`
+This release has the following deprecated feature:
```
Comment (anarnold97) [inline documentation/modules/rn-2.5.adoc:198]: @ahadas is this ok to cover - [MTV-727 - deprecated feature](https://issues.redhat.com/browse/MTV-727)

image

Comment (anarnold97) [reply documentation/modules/rn-2.5.adoc:199]: removed

Comment (ahadas) [reply documentation/modules/rn-2.5.adoc:199]: @anarnold97 in #434 you added a screenshot of the modified note but where was it implemented? I don't see it here or in #434 
and we dropped the 'deprecated' part, so do we really need to include it in the release notes?..

Comment (ahadas) [reply documentation/modules/rn-2.5.adoc:199]: ah ok, I got it - it's commented out...

**Pull request merged**